pkg/runtime/runtime.go

```diff
@@ -1264,8 +1213,8 @@ func (a *DaprRuntime) sendBindingEventToApp(bindingName string, data []byte, met
 	ctx, span := diag.StartInternalCallbackSpan(a.ctx, spanName, spanContext, a.globalConfig.Spec.TracingSpec)
 
 	var appResponseBody []byte
-	path := a.inputBindingRoutes[bindingName]
-	if path == "" {
+	path, ok := a.compStore.GetInputBindingRoute(bindingName)
+	if !ok || path == "" {
```

Comment:
Nit: if the property doesn't exist, it returns the zero value which is `""`. So checking `ok` is redundant 

```suggestion
	path := a.compStore.GetInputBindingRoute(bindingName)
	if path == "" {
```